src/components/header/renderer.tsx

```diff
@@ -116,7 +120,9 @@ class Header extends React.PureComponent<Props, State> {
   }
 
   public onSwitchMode(option) {
-    option.value === Mode.Vega ? this.props.updateVegaSpec(stringify(this.props.vegaSpec)) : this.onSelectNewVegaLite();
```

Comment:
Can you replace this with an if block instead of a ternary?

Reply:
yes, I will replace it with if-else block

Reply:
Thanks